src/acl/external/session/ext_session_acl.cc

```diff
@@ -202,8 +202,8 @@ static int session_active(const char *details, size_t len)
     key.data = (void *)details;
     key.size = len;
 #elif USE_TRIVIALDB
-    TDB_DATA key;
-    TDB_DATA data;
+    TDB_DATA key = {};
+    TDB_DATA data = {};
     (void)len;
```

Comment:
AFAICT, Coverity found a huge bug and this PR hides that bug from Coverity without fixing the problem. Please double check that our fetchKey() and library [tdb_fetch](https://manpages.debian.org/testing/libtdb-dev/tdb_fetch.3.en.html)() it calls are indeed meant to be called with zero-initialized key, leaving `details` unused. It makes no sense to me.

```C++
fetchKey(/*const*/ DB_ENTRY &key, DB_ENTRY *data)
{
    ...
    *data = tdb_fetch(db, key); // "Fetch the record from the tdb database whose key matches key"
    ...
}
```

Reply:
You are right ,and this defect highlights a gaping bug, where the key we are looking up is never initialised in the tdb case.
There is also the potential optimization of initialising the data members of the key only once.

Regarding tdb_fetch, the struct is returned, so I expect that the implicit assignment operator will take care of thigns. We are initialising the struct twice but it won't hurt

Next issue: at least in ubuntu impish, the implementation of TDB_DATA doesn't match the documentation - it's an unsigned char * . 

Checking and iterating

Reply:
Okay, this compiles despite the mismatch with documentation